src/java.desktop/macosx/classes/sun/font/CCompositeGlyphMapper.java

```diff
@@ -51,10 +51,28 @@ public boolean canDisplay(char ch) {
         return glyph != missingGlyph;
     }
 
-    private int convertToGlyph(int unicode) {
+    @Override
+    public int charToVariationGlyph(int unicode, int variationSelector) {
+        if (variationSelector == 0) return charToGlyph(unicode);
+        else {
+            int glyph = convertToGlyph(unicode, variationSelector);
+            if (glyph == missingGlyph) glyph = charToGlyph(unicode);
+            return glyph;
+        }
+    }
+
+    @Override
+    protected int convertToGlyph(int unicode) {
+        int glyph = convertToGlyph(unicode, 0);
+//        setCachedGlyphCode(unicode, glyph);
```

Comment:
What's the purpose of the commented line?

Reply:
Committed this accidentally, should have deleted this line